quiz-template.html

```diff
@@ -1,148 +1,185 @@
-<div class="questions">
-Question #1
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question1"> a) Description <br>
-<input type ="radio" value ="y" name="question1"> b) Description <br>
-<input type ="radio" value ="z" name="question1"> c) Description <br>
-
-</div>
-
-<br>
-
-<div class="questions">
-Question #2
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question2"> a) Description <br> 
-<input type ="radio" value ="y" name="question2"> b) Description <br>
-<input type ="radio" value ="z" name="question2"> c) Description <br>
-</div>
-
-<br>
-
-<div class="questions"">
-Question #3
-</div>
-
-<div class="answers">
-<input type ="radio" value ="x" name="question3"> a) Description <br>
-<input type ="radio" value ="y" name="question3"> b) Description <br>
-<input type ="radio" value ="z" name="question3"> c) Description <br> 
-</div>
-
-<br>
-
-</form>
-
-<!-- Submit button -->
-<form>
-<input type = "button" value="Submit!" name ="submit" onClick="displayResult()">
-</form>
-
-</div> 
+          }
+        }
+      }
+
+      /* Generates the text to appear in the results */
+      function generateCategory() {
+        if (xTotal > Math.max(yTotal, zTotal)) { // category x
+          categoryName = "X";
+          categoryDescription = "Supplementary information for x :3";
+        } else if (yTotal > Math.max(xTotal, zTotal)) { // category y
+          categoryName = "Y";
+          categoryDescription = "Supplementary information for y :3";
+        } else if (zTotal > Math.max(xTotal, yTotal)) { // catgory z
+          categoryName = "Z";
+          categoryDescription = "Supplementary information for z :3";
+        } else { // if ties are possible or if no questions are answered
+          categoryName = "Any";
+          categoryDescription = "Hm maybe you should retake the quiz? :3";
+        }
+      }
+
+      /* TODO: 
+         - Check whether the user answered all questions
+           for browsers that don't support the "required" attribute.
+       */
+      function displayResult() {
+        grade();
+        generateCategory();
+        hideMain();
+
+        // fill in fields
+        document.getElementById("result-category").innerHTML = categoryName;
+        document.getElementById("result-description").innerHTML = categoryDescription;
+
+        // display result
+        document.getElementById("result").style.display = "block";
+      }
+  </script>
+</head>
+
+<body>
+
+  <!-- Main -->
+  <div id="quiz-main">
+
+    <h1>Quiz Title</h1>
+
+    <!-- Form -->
+    <form id="quiz" onsubmit="displayResult(); return false;">
+      <!-- Questions. NOTE: To avoid ties, choose 4, 7, 10, etc. questions
+           Remove the "required" attribute to make a question optional -->
+
+      <fieldset class="question">
+        <legend>Question #1</legend>
+        <label><input type="radio" value="x" name="question1" required> a) Description</label><br>

```

Comment:
Added the `required` attribute so that the browser will complain when it's left blank.
I would usually do `required="required"` in full, but since you're using the HTML syntax and not the XHTML syntax, this is acceptable.